x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/advanced_step/hyper_parameters.tsx

```diff
@@ -192,8 +190,7 @@ export const HyperParameters: FC<Props> = ({ actions, state, advancedParamErrors
             defaultMessage: 'Randomize seed',
           })}
           helpText={i18n.translate('xpack.ml.dataframe.analytics.create.randomizeSeedText', {
-            defaultMessage:
-              'The seed to the random generator that is used to pick which documents will be used for training.',
+            defaultMessage: 'The seed for the random generator that picks training data.',
```

Comment:
nit: this reads slightly better to me because it isn't strictly the rng which is picking the data
```suggestion
            defaultMessage: 'The seed for the random generator used to pick training data.',
```